src/components/GenomeFeature.vue

```diff
@@ -1,29 +1,28 @@
 <template>
-  <div
+  <a
```

Comment:
It's unclear to me why you are turning the SVG into an anchor. Is it so we can navigate to this section? If so, it is better to have the anchor be separate from the content.

Reply:
@DoctorBud you should have a look at the code.  Its just swapping the anchor and svg sections and removing the unused jbrowse-link id.

Reply:
Can you point me to the code you're talking about? None of this explains making the whole section an anchor (`<a>`). I want to believe! My current hypothesis is that there is a mistake in your model of the world or mine, and it would be nice to resolve. Thanks for your patience.